topics/xampp/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: xampp-server, xampp-control-panel
+created_by: Apache Friends
+display_name: XAMPP
+logo: xampp.png
+released: 2002
+short_description: XAMPP is the combination of the Apache web server, the MySQL database and scripting langs.
+topic: xampp
+url: https://www.apachefriends.org/de/index.html
```

Comment:
Rather than link to the `/de/` page directly, should we link to the root `index.html` and let the server handle redirection to the correct localized content?

```suggestion
url: https://www.apachefriends.org/index.html
```